tests/osaka/eip7692_eof_v1/eip7069_extcall/test_calls.py

```diff
@@ -57,12 +58,67 @@
 )
 
 
+@unique
+class TargetAccountType(Enum):
+    """Kinds of target accounts for calls."""
+
+    EMPTY = auto()
+    EOA = auto()
+    LEGACY_CONTRACT = auto()
+    EOF_CONTRACT = auto()
+    LEGACY_CONTRACT_INVALID = auto()
+    EOF_CONTRACT_INVALID = auto()
+    LEGACY_CONTRACT_REVERT = auto()
+    EOF_CONTRACT_REVERT = auto()
+    IDENTITY_PRECOMPILE = auto()
+
+    def __str__(self) -> str:
+        """Return string representation of the enum."""
+        return f"{self.name}"
+
+
 @pytest.fixture
 def sender(pre: Alloc) -> EOA:
     """Sender of the transaction."""
     return pre.fund_eoa()
 
 
+@pytest.fixture
+def target_address(pre: Alloc, target_account_type: TargetAccountType) -> Address:
+    """Target address of the call depending on required type of account."""
+    match target_account_type:
+        case TargetAccountType.EMPTY:
+            return Address(b"\x78" * 20)
```

Comment:
```suggestion
            return pre.fund_eoa(amount=0)
```
This will guarantee that an empty account is used on any network, and that we can re-execute the same test on a live-network, because, e.g. if we run the same test twice in a devnet, on the second run of the same test the account will already be touched by the previous run of the test case.